docs/blog/posts/2023/nanog88-python-pygnmi.md

```diff
@@ -0,0 +1,99 @@
+| clab-dc-k8s-LEAF-DC-2 |    l2evpn1001    | 2  |      enable      |   vxlan2.1001   | 1001 |  1   |     up     | 1.1.1.2:1001 | target:65123:1001 | target:65123:1001 |
+| clab-dc-k8s-LEAF-DC-1 |    l2evpn1002    | 2  |      enable      |   vxlan2.1002   | 1002 |  1   |     up     | 1.1.1.1:1002 | target:65123:1002 | target:65123:1002 |
+| clab-dc-k8s-LEAF-DC-2 |    l2evpn1002    | 2  |      enable      |   vxlan2.1002   | 1002 |  1   |     up     | 1.1.1.2:1002 | target:65123:1002 | target:65123:1002 |
+| clab-dc-k8s-LEAF-DC-1 |    l2evpn1003    | 2  |      enable      |   vxlan2.1003   | 1003 |  1   |     up     | 1.1.1.1:1003 | target:65123:1003 | target:65123:1003 |
+| clab-dc-k8s-LEAF-DC-2 |    l2evpn1003    | 2  |      enable      |   vxlan2.1003   | 1003 |  1   |     up     | 1.1.1.2:1013 | target:65123:1013 | target:65123:1013 |
+| clab-dc-k8s-LEAF-DC-1 |    l2evpn1004    | 2  |      enable      |   vxlan2.1004   | 1004 |  1   |     up     | 1.1.1.1:1004 | target:65123:1004 | target:65123:1004 |
+| clab-dc-k8s-LEAF-DC-2 |    l2evpn1004    | 2  |      enable      |   vxlan2.1004   | 1004 |  1   |     up     | 1.1.1.2:1004 | target:65123:1004 | target:65123:1004 |
+| clab-dc-k8s-LEAF-DC-1 |    l2evpn1005    | 2  |      enable      |   vxlan2.1005   | 1005 |  1   |     up     | 1.1.1.1:1005 | target:65123:1005 | target:65123:1005 |
+| clab-dc-k8s-LEAF-DC-2 |    l2evpn1005    | 2  |      enable      |   vxlan2.1005   | 1005 |  1   |     up     | 1.1.1.2:1005 | target:65123:1005 | target:65123:1005 |
+| clab-dc-k8s-LEAF-DC-1 |    l2evpn1006    | 2  |      enable      |   vxlan2.1006   | 1006 |  1   |     up     | 1.1.1.1:1006 | target:65123:1006 | target:65123:1006 |
+| clab-dc-k8s-LEAF-DC-2 |    l2evpn1006    | 2  |      enable      |   vxlan2.1006   | 1006 |  1   |     up     | 1.1.1.2:1006 | target:65123:1006 | target:65123:1006 |
+| clab-dc-k8s-LEAF-DC-1 |      l3evpn      | 1  |      enable      |    vxlan1.2     |  2   |  4   |     up     |  1.1.1.1:2   |  target:65123:2   |  target:65123:2   |
+| clab-dc-k8s-LEAF-DC-2 |      l3evpn      | 1  |      enable      |    vxlan1.2     |  2   |  4   |     up     |  1.1.1.2:2   |  target:65123:2   |  target:65123:2   |
++-----------------------+------------------+----+------------------+-----------------+------+------+------------+--------------+-------------------+-------------------+
+Total time: 1.42 seconds
+```
+Now, if you have a typo in the EVI number, then, I added a function will show you that:
+![EVPN configuration typo](https://github.com/cloud-native-everything/pygnmi-srl-nanog88/blob/main/py-scripts/images/Highligthed-Typo-EVPN-Fabric-Configuration.png)
+
+## How to use it
+To use the python class, you'll need to install some modules, including tabulate and pygnmi. Following this, you can import the class as such:
+```python
+from srl_evpn_class import SrlDevice
+from srl_evpn_class import MergeEvpnToArray
+from srl_evpn_class import HighlightAlternateGroups
```

Comment:
this doesn't look like idiomatic python code. You generally want to call the class just as `Srlinux` or `SrlEvpn`.

Reply:
done